.github/workflows/test.yml

```diff
@@ -60,29 +54,16 @@ jobs:
 
   # Run acceptance tests in a matrix with Terraform CLI versions
   test:
-    name: Terraform Provider Acceptance Tests
+    name: Unit Test
     needs: build
     runs-on: ubuntu-latest
-    timeout-minutes: 15
-    strategy:
-      fail-fast: false
-      matrix:
-        # list whatever Terraform versions here you would like to support
-        terraform:
-          - "1.10.*"
```

Comment:
This removes the matrix from this job, fixes the two jobs above it to not require a matrix, but retains the matrix in the new `main-test.yml` -- should these jobs all work the same?

Reply:
I don't think it really matters much either way TBH.  I left it as optional for the acceptance tests because it seemed more important there but TBH I'd rather just avoid the matrix since it screws up "check" names.

Reply:
Sounds good, if we need the matrix in the future it's easy enough to add.